### DISCUSSION TITLE
deep-translator

### DISCUSSION THREAD

**Discussion by fionaeliza** [nidhaloff/deep-translator#109] [Q&A]
Hi all, I just wanted to ask about data security when using this translator. Could anyone elaborate on how secure it is to be using this? Please excuse my ignorance on this topic.

**Comment by nidhaloff** [comment]
Hi @fionaeliza, If you mean whether deep-translator stores your data in any form, then the answer is no

**Reply by bhuvanesh-ctrl** [reply]
And what about the supporting translators? If I am using google translator through deep-translator is my data going and sitting in google cloud or is it just hitting the servers, processing the input and providing an output?
